ref/config/serverConfiguration.adoc

```diff
@@ -318,18 +260,11 @@ is treated as:
 </server>
 ----
 
-==== Merging factory config
-
-Factory config merges use the same rules as singleton config but, because
-it is valid to configure the same element and mean two different logical objects,
-merging doesn't happen just because the element names match. Instead each
-element is assumed to be configuring a distinct object. If the logical object is
-configured by two instances, the `id` attribute must be set on each of them
-to indicate they are the same thing. Variable substitution on an `id` is not
-supported.
+=== Merging factory configuration
+Factory config merges use the same rules as Singleton config except elements are not automatically merged just because the names match. Unlike Singleton config, with factory config it is valid to configure the same element and mean two different logical objects. Therefore, each element is assumed to configure a distinct object. If the logical object is configured by two instances, the `id` attribute must be set on each of them to indicate they are the same thing. Variable substitution on an `id` attribute is not supported.
```

Comment:
singleton.

it says not merged just because the names matched, but it doesn't specify what a name is. In this case it means the element name, but I think that needs to be clarified other wise it could be inferred that it means the name attribute.

This reads incorrectly vs the actual behaviour:

>If the logical object is configured by two instances, the `id` attribute must be set on each of them to indicate they are the same thing.

You are saying to have two elements configure the same object you need to use the id attribute with the same value, but the way it reads with reference to two instances as expressed it makes it sound like you have to specify the id attribute if you want 2 logical instances, rather than one.

Reply:
This wasn't changed from the original:
> > If the logical object is configured by two instances, the `id` attribute must be set on each of them to indicate they are the same thing.

Would it be better as:
If a logical object is configured by two elements, the `id` attribute must be set on each element to indicate they are the same thing.

Reply:
sorry I had missed the original error was mine not yours. What about:

> If a single logical object is configured by two elements, the id attribute must be set on each element to indicate they are the same thing.

Reply:
Updated.